src/main/java/com/glencoesoftware/omero/zarr/ZarrPixelBuffer.java

```diff
@@ -307,6 +322,37 @@ public List<Map<String, String>> getDatasets() {
                 getMultiscalesMetadata().get(0).get("datasets");
     }
 
+    /**
+     * Retrieves the axes metadata of the first multiscale
+     * @return See above.
+     */
+    public Map<Axis, Integer> getAxesOrder() {
```

Comment:
I feel like this should either be `setAxesOrder` and set the member variable from the zarr, or it should check to see if the value has been set and just return `axesOrder` if it has.